great_expectations/rule_based_profiler/parameter_builder/simple_date_format_string_parameter_builder.py

```diff
@@ -65,6 +117,7 @@ def __init__(
         self._metric_value_kwargs = metric_value_kwargs
 
         self._threshold = threshold
+
```

Comment:
In this code, we have tuple instead of dictionary -- extra parentheses must be removed.
```
                match_strftime_metric_value_kwargs: dict = (
                    {
                        **self._metric_value_kwargs,
                        **{"strftime_format": fmt_string},
                    },
                )
```
@NathanFarmer -- thanks.  Also, stylistically, comma after each dictionary key/value pair, even if the only one.  Thanks!